lte/gateway/release/magma-postinst

```diff
@@ -37,6 +37,8 @@ systemctl stop lighttpd
 systemctl disable lighttpd
 
 # Restart rsyslog to pick up fluent-bit config, create fluent-bit DB directory
+cp /etc/logrotate.d/rsyslog /etc/logrotate.d/rsyslog.orig
```

Comment:
I think using apt with force overwrite is less hacky than that. It's already in the upgrade process.

Reply:
Releasing a package with known conflicts is not good practice. There are two methods to handle config file in debian packages. Using post-install script is one of them. since we are already using post-install script, I decided to extend it.
https://www.debian.org/doc/debian-policy/ap-pkg-conffiles.html

Having said that, I dont feel strongly enough about this. so I am fine with dropping this PR.